views/proposals.tmpl

```diff
@@ -118,14 +120,28 @@
                     <tr>
                         <td class="text-left"><a href="/proposal/{{.RefID}}">{{.Name}}</a></td>
                         <td class="text-left">{{.Username}}</td>
-                        {{with .Status}}
-                            <td class="text-left">{{toTitleCase .String}}</td>
-                        {{end}}
                         <td class="text-left">
-                            {{if .VoteStatus}}{{with .VoteStatus}}{{.ShortDesc}}{{end}}{{else}}N/A{{end}}
+                            {{if .VoteStatus}}
+                                {{range $i, $vr := .VoteResults}}
+                                    {{if eq $vr.Option.OptionID "yes"}}
+                                        {{$percent :=  percentage $vr.VotesReceived $v.TotalVotes}}
+                                        {{if lt $percent 60.0}}
+                                            <span style="color:#ED6D47;">Failed
+                                        {{else}}
+                                            <span style="color:#2DD8A3;">Passed
+                                        {{end}}
+                                        ({{printf "%.0f" ($percent)}}%)</span>
```

Comment:
the way the close of the `span` intertwines with the `if` statement feels funny but it has produced the desired effect

Reply:
I agree that its a little funny looking, but it works. If you can improve the readability, feel free.

Reply:
You need to add the `InProgress(% approval)` option whereby `proposalInfo.ProposalVotes.Endheight` will help know the last block when voting should be complete after which `Failed(% approval)` or `Passed(% approval)` status can be displayed.